_posts/2017-10-23-survey-focus-group-update.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+subheadline: "Assessment"
+title: "Pilot Survey Focus Group Update"
+teaser: "Outcomes from initial focus group to discuss Data Carpentry’s survey instruments"
+header:
+    image_fullwidth: "light-blue-wood-texture.jpg"
+categories:
+    - blog
+comments: true
+show_meta: true
+authors: ["Kari L. Jordan", “Stephen Childs”, “Alycia Crall", “Reshama Shaikh”, “Aleksandra Nenadic”, “Karen Word”, “Louisa Bells”]
+---
+
+
+In April 2017, Data Carpentry began to pilot a [new survey](https://www.surveymonkey.com/r/presurveyfocusgroup?workshop_id=[workshop_id_value]) that includes skills-based questions. Check out the [blog post](http://www.datacarpentry.org/blog/long-term-assessment-strategy/) about our assessment strategy, and the work involved leading up to this effort.
+
+The survey includes the following statement:
+
+        Please enter a unique identifier as follows: Number of siblings (as numeric) + First two letters of the city you were       
+        born in (lowercase) + First three letters of your current street (lowercase). This identifier will be confidential to 
+        you and will help us pair your results with the post-assessment.
+ 
+        Example: If I have 4 siblings, was born in Arlington, and live on Creekwater Street, my unique identifier would be 
+        4arcre
+
+This unique identifier is asked of respondents so that our staff can run [paired analyses](http://www.statstutor.ac.uk/resources/uploaded/paired-t-test.pdf) of the pre- and post-survey results to measure significant gains post-workshop.
+
+In addition to the unique identifier, the survey includes skills-based questions. One example of a survey question is:
+
+        ggplot is an R package that is used to build plots from data in a dataframe. If ‘df’ is your dataframe and has 
+        columns x and y, which of the following lines of code will produce a plot of x versus y?
+          A: ggplot <- df
+          B: ggplot(df)
+          C: ggplot(df, aes(x, y))
+          D: ggplot(df, aes(x, y)) + geom_point()
+          E. None of the above will work.
+
+Our goal in including skills-based questions was to be able to show that learners were leaving our workshops having specific skills working with data.
+
+Feedback from our instructor community has been that this survey has given some of our learners anxiety about our workshops. Though language around the purpose of the unique identifier was included, some learners feel the questions they are being asked are intrusive, and that the skills-based questions are intimidating. 
```

Comment:
"our instructor community" --> "some of our instructors" (to make it clear this isn't a widespread problem)

"feel/are" --> "felt/were" (I prefer past tense phrasing here as it help with the narrative of "identifying a problem and fixing it")